contrib/init/datum_gateway.service

```diff
@@ -0,0 +1,89 @@
+[Unit]
+Description=DATUM Gateway
+Documentation=https://github.com/OCEAN-xyz/datum_gateway
+
+#
+# Expected file locations when using hardening options:
+# binary: /usr/bin/datum/
+# configuration: /etc/datum/
+# logs: /var/lib/datum_gateway/
+#
+
+# https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
+Wants=network-online.target
+After=network-online.target
+# alternatively, wait until bitcoind is up
+#After=bitcoind.service
```

Comment:
Currently, datum_gateway will fail to start if bitcoind isn't responsive - so maybe this should be uncommented?

Reply:
Well, it *starts* if bitcoind is unresponsive. It just doesn't do anything useful, except log a lot of errors. But the reason for leaving `After=bitcoind.service` line uncommented was simply that someone might be configuring datum to run via systemd without configuring bitcoind to run via systemd. But could leave it uncommented and have the comment say instead to remove it if bitcoind isn't configured via systemd.